tests/conftest.py

```
@@ -169,6 +169,12 @@ def is_postcopy_migration_bug_open(cluster_has_rhcos10_or_above):
     return cluster_has_rhcos10_or_above and is_jira_open(jira_id="CNV-84023")
 
 
+@pytest.fixture()
```

Ruff (0.16.2)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)